core/nwb.ecephys.yaml

```
@@ -257,6 +257,75 @@ groups:
     target_type: Device
     doc: Link to the device that was used to record from this electrode group.
 
+- neurodata_type_def: ElectrodesTable
+  neurodata_type_inc: DynamicTable
+  doc: A table of all electrodes (i.e. channels) used for recording. Introduced in NWB 3.0.0. Replaces the "electrodes"
```

```suggestion
  doc: A table of all electrodes (i.e. channels) used for recording. Introduced in NWB 2.8.0. Replaces the "electrodes"
```

The next release is 2.8.0, not 3.0.0